rust/src/mask/config.rs

```diff
@@ -244,405 +305,407 @@ impl MaskConfigs {
                     }
                     B2 => match self.model_type {
-                        M3 => BigUint::from_str_radix("262_144", 10).unwrap(),
-                        M6 => BigUint::from_str_radix("268_435_456", 10).unwrap(),
-                        M9 => BigUint::from_str_radix("274_877_906_944", 10).unwrap(),
-                        M12 => BigUint::from_str_radix("281_474_976_710_656", 10).unwrap(),
+                        M3 => "262_144",
+                        M6 => "268_435_456",
+                        M9 => "274_877_906_944",
+                        M12 => "281_474_976_710_656",
                     }
                     B4 => match self.model_type {
-                        M3 => BigUint::from_str_radix("33_554_432", 10).unwrap(),
-                        M6 => BigUint::from_str_radix("34_359_738_368", 10).unwrap(),
-                        M9 => BigUint::from_str_radix("35_184_372_088_832", 10).unwrap(),
-                        M12 => BigUint::from_str_radix("36_028_797_018_963_968", 10).unwrap(),
+                        M3 => "33_554_432",
+                        M6 => "34_359_738_368",
+                        M9 => "35_184_372_088_832",
+                        M12 => "36_028_797_018_963_968",
                     }
                     B6 => match self.model_type {
-                        M3 => BigUint::from_str_radix("2_147_483_648", 10).unwrap(),
-                        M6 => BigUint::from_str_radix("2_199_023_255_552", 10).unwrap(),
-                        M9 => BigUint::from_str_radix("2_251_799_813_685_248", 10).unwrap(),
-                        M12 => BigUint::from_str_radix("2_305_843_009_213_693_952", 10).unwrap(),
+                        M3 => "2_147_483_648",
+                        M6 => "2_199_023_255_552",
+                        M9 => "2_251_799_813_685_248",
+                        M12 => "2_305_843_009_213_693_952",
                     }
                     Bmax => match self.model_type {
-                        M3 => BigUint::from_str_radix("18_889_465_931_478_580_854_784", 10).unwrap(),
-                        M6 => BigUint::from_str_radix("19_342_813_113_834_066_795_298_816", 10).unwrap(),
-                        M9 => BigUint::from_str_radix("19_807_040_628_566_084_398_385_987_584", 10).unwrap(),
-                        M12 => BigUint::from_str_radix("20_282_409_603_651_670_423_947_251_286_016", 10).unwrap(),
+                        M3 => "18_889_465_931_478_580_854_784",
+                        M6 => "19_342_813_113_834_066_795_298_816",
+                        M9 => "19_807_040_628_566_084_398_385_987_584",
+                        M12 => "20_282_409_603_651_670_423_947_251_286_016",
                     }
                 }
             }
         };
-        MaskConfig {
-            name,
-            add_shift,
-            exp_shift,
-            order,
-        }
+        BigUint::from_str_radix(order_str, 10).unwrap()
```

Comment:
the safe unwrap comment was deleted, sth like `// safe unwrap: string and radix are valid` should probably be placed in front of this line.

Reply:
Ah yes sorry.